solutions/gold/usaco-993.mdx

```diff
@@ -22,48 +32,54 @@ We only need to keep track of the number of days that Bessie has been traveling
 #include <bits/stdc++.h>
 using namespace std;
 
-const int MAX_DAYS = 1000;
-
 int main() {
 	freopen("time.in", "r", stdin);
 	freopen("time.out", "w", stdout);
 
 	int n, m, c;
 	cin >> n >> m >> c;
 
-	vector<int> earn(n);
-	for (int i = 0; i < n; i++) { cin >> earn[i]; }
+	vector<int> moonies(n);
+	for (int i = 0; i < n; i++) { cin >> moonies[i]; }
 
 	vector<vector<int>> adj(n);
 	for (int i = 0; i < m; i++) {
-		int u, v;
-		cin >> u >> v;
-		adj[--u].push_back(--v);
+		int a, b;
+		cin >> a >> b;
+		a--, b--;  // Convert 1-indexed input to 0-indexed
+		adj[a].push_back(b);
 	}
 
-	// dp[i][j] = the max money that Bessie can make on day i if she ends in
-	// city j
-	vector<vector<int>> dp(MAX_DAYS + 1, vector<int>(n, -1));
-	// base case: if Bessie doesn't travel at all, she makes $0
-	dp[0][0] = 0;
+	const int MAX_DAYS = 1000;  // Maximum number of days Bessie can travel
```

Comment:
any reason you moved this inside?

Reply:
fwiw he told me that he will be working more on the other PR he made

Reply:
> any reason you moved this inside?

no real reason, just defined it and forgot to put it outside, will do that now